obal/data/roles/git_branch_and_commit/tasks/main.yml

```diff
@@ -0,0 +1,29 @@
+---
+- name: 'Get updated version from spec'
+  command: "rpmspec --query --queryformat=%{version} --srpm {{ spec_file_path }}"
+  args:
+    chdir: "{{ inventory_dir }}"
+  changed_when: true
```

Comment:
I'd think this shall be `false` as a query should never *change* something (and it was false in the olde code)